cmd/auto-updater/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/internal/sync2"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "auto-updater",
+		Short: "auto-updater for storage node",
```

Comment:
Should this start with a capital letter? i.e. `Auto-updater for storage node`

Reply:
Yes, it looks that in other commands we have a capital letter.